### DISCUSSION TITLE
V2024.11.1

### DISCUSSION THREAD

**Discussion by WatsonYao** [WatsonYao/nian_app#111] [version]
●新增「什物集」，可简单分类并整合日常用品的使用信息。
●优化部分页面细节，修复部分用户反馈过来的问题，提升性能和稳定性。
首页搜索范围添加倒序按钮。
心情集统计页支持年份切换。
从桌面小组件/快捷方式进入的集子和记本，点击左上角箭头，改为跳转首页。
桌面小组件的文字使用纯白或纯黑。
注意：有在使用桌面小组件或快捷方式的，安装新版本后需要重启下手机。
![捕获](https://github.com/user-attachments/assets/08ae9b08-5237-4c70-bb79-ba93e2634535)

<hr /><em>This discussion was created from the release <a href='https://github.com/WatsonYao/nian_app/releases/tag/V2024.11.1'>V2024.11.1</a>.</em>

**Comment by Elegy17** [comment]
## 功能越来越多了，超爱这软件🤤

**Comment by dustwa** [comment]
关于什物集的反馈

1. 也许输入框的文字和顺序可以改成这样? 
   上下对齐起始和终止日还是有点反直觉.
   文字也可以再缩短一点
  生产日 到期日
  生效日 失效日
2. 价格和tag等信息可以从账本中`相互`链接,不然就又要输入价格了.同理,也可以和点评集链接
3. 增加记录元数据相关的部分.
 有别于`备注`,元数据不会被展示出来
 如记录MEID,条形码,药物的使用剂量等,虽然tag可以记录,但不能按`关键字`来筛选(尽管可以像这样做`尺寸:14`),但我不希望元数据被展示出来,特别是`条形码`这种长串字符.

另外我还希望进度集也可以和点评集链接,书本读完后就把进展链接到点评集进行评分,而不是默认留个10分看完后再去改动.

**Reply by WatsonYao** [reply]
1. 上面日期更新一些，关注的多是截止和结束日期，还方便上下对齐做减法看差值。文案暂时不做更改了，字数差异故意加的，怕看成横行一组。
2. 标签不做链接。
3. 不展示的数据，这块确实没有人提到，看后续反馈吧，提到这点多的话加个隐藏备注的开关。tag都是可以筛选和搜索的。

关于各类集子和记本的互联问题，还在观望中。这里没有统一的场景，近十类数据形式互联，一旦做的复杂，软件编码和用户使用负担都是陡增，性价比很低。可能会尝试如何做的简单，比如一个主题，添加相关记本或集子，提供近期内容展示之类的。

**Reply by dustwa** [reply]
3. 我一开始的想法是对每个tag提供一个显示或隐藏的选项,这样的话其他计本也可以使用,不然就太滥用备注了.
啊知道了,可以这样`tag0 --- tag1 tag2`,把`---`后面的tag都隐藏掉,这样操作起来就没那么繁琐了.

也就是像记账本那样左右切换视图吗?不过我更偏向于类似手机桌面的小部件那样可以组合各个部件,布局更灵活点.当然全都有最好.
既然如此,在创建主题的时候也可以把关联数据的工作交给用户,只要数据的类型相同就可以进行关联,这样代码就不用写死了.
而关联界面则类似基本`计本分组`,使用时,当自身属于链接出的数据的时候默认隐藏自己.
